shoop/campaigns/models/catalog_filters.py

```diff
@@ -19,6 +20,9 @@ class CatalogFilter(PolymorphicModel):
 
     active = models.BooleanField(default=True, verbose_name=_("active"))
 
+    def __str__(self):
+        return force_text(self.name)
```

Comment:
Name of the filter class in not very good string for the object, since str of the object should have the values of the object too (e.g. "Product type: Normal product" or "Product: SOME-SKU-123", not just "Product type" or "Product").  Why are you defining str here?